src/util/MusicSingleton.js

```diff
@@ -125,9 +125,11 @@ class MusicSingleton {
   }
 
   stop(message) {
-    if (!message.member.voice.channel) {
+    if (message && message.member.voice.channel) {
       return message.reply('Please join voice channel first!');
     }
```

Comment:
assuming this passes lint
```suggestion
    if (!message) return;
    if (!message.member.voice.channel) {
      return message.reply('Please join voice channel first!');
    }
```